bundles/io/org.openhab.io.caldav/src/main/java/org/openhab/io/caldav/internal/CalDavConfig.java

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.io.caldav.internal;
+
+/**
+ * Configuration class for binding configuration settings.
+ * 
+ * @author Robert Delbrück
+ * @since 1.8.0
+ */
+public class CalDavConfig {
+	private String key;
+	private String username;
+	private String password;
+	private String url;
+	private int reloadMinutes = 60;
+	private int preloadMinutes = 60 * 24;
+	private int historicLoadMinutes = 0;
+	private boolean disableCertificateVerification;
+	
+	public CalDavConfig() {
+		super();
```

Comment:
CalDavConfig has no super class (except of java.lang.Object). Do we need a super() here?